core/src/main/java/me/uport/sdk/core/Networks.kt

```diff
@@ -31,7 +30,8 @@ object Networks {
                 rpcUrl = "https://mainnet.infura.io/v3/e72b472993ff46d3b5b88faa47214d7f",
```

Comment:
Some of the previous code has been commented out rather than deleted (e.g `MNID.encode(mainnetId, "0xab5c8051b9a1df1aab0149f8b0630848b7ecabf6")`. Is this intentional?

Reply:
yes, it's intentional to leave a mark of the addresses of the registries